test/CMakeLists.txt

```diff
@@ -35,13 +35,16 @@ set(tests
   player_test
   quests_test
   random_test
-  timedemo_test
   scrollrt_test
   stores_test
   utf8_test
   writehero_test
 )
 
+if(NOT DISABLE_DEMOMODE)

```

Comment:
Could we make it incompatible with building tests instead?

Reply:
Done, now setting `DISABLE_DEMOMODE` disables `BUILD_TESTING`.